bin/desi_job_graph

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python
+
+"""
+Create a webpage with the dependency graph and status for DESI jobs on a night
+"""
+
+import os, sys, glob
+import numpy as np
+from astropy.table import Table
+
+import argparse
+
+from desispec.workflow.proctable import get_processing_table_path
+from desispec.workflow.proctable import get_processing_table_name
+from desispec.workflow.queue import queue_info_from_qids
+from desispec.io.meta import specprod_root
+
+p = argparse.ArgumentParser()
+p.add_argument('-n', '--night', type=int, required=True,
+        help='night to process')
+p.add_argument('-o', '--output', type=str, required=False,
+        help='output HTML file (default run/jobgraph/jobgraph-NIGHT.html in reduxdir)')
+p.add_argument('-s', '--specprod', type=str, required=False,
+        help=('override $SPECPROD, or full path '
+              'to override $DESI_SPECTRO_REDUX/$SPECPROD'))
+
+args = p.parse_args()
+
+#- Default output file into specprod directory
+if args.specprod is not None and os.path.isdir(args.specprod):
+    reduxdir = args.specprod
```

Comment:
Could we title this `specproddir` or `specdir` if it includes the `specprod`?

Reply:
done